atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -972,8 +973,13 @@ private static LockAndTimestampServices getLockAndTimestampServices(
             Optional<ClientLockDiagnosticCollector> lockDiagnosticCollector) {
         ServiceCreator creator = ServiceCreator.withPayloadLimiter(
                 metricsManager, timelockServerListConfig, userAgent, remotingConfigSupplier);
-        LockService lockService = RemoteLockServiceAdapter.create(
-                creator.createService(LockRpcClient.class), timelockNamespace);
+
+        // I'd use instrumentTimed(), but the timed annotations are not visible in LockService. Also, most practical
```

Comment:
I think you'll merge conflict with #4404 no?

Reply:
Or once #4404 merges, just change it here? no harm in it really

Reply:
This shouldn't actually conflict, just that this won't use the smart features in #4404

Reply:
Switched this call to `instrumentTimed()`.